src/main/java/jewellery/inventory/controller/ResourceController.java

```diff
@@ -0,0 +1,53 @@
+package jewellery.inventory.controller;
+
+import jakarta.validation.Valid;
+import java.util.List;
+import java.util.UUID;
+import jewellery.inventory.dto.request.resource.ResourceRequestDto;
+import jewellery.inventory.dto.response.resource.ResourceResponseDto;
+import jewellery.inventory.service.ResourceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/resources")
+@RequiredArgsConstructor
+public class ResourceController {
```

Comment:
Maybe we should put
@CrossOrigin(origins = "${cors.origins}") 
I saw you added it to the home controller but not here